### ISSUE TITLE
Error Log spam

### ISSUE THREAD

**Issue by Phoupraw** [skniro/GrowableOres#19]
```
[Render thread/ERROR]: Parsing error loading recipe growable_ores:ic2c/ic_tin_ore_cane
com.google.gson.JsonParseException: Map entry 'G' : Failed to parse either. First: Input does not contain a key [fabric:type]: MapLike[{"item":"ic2:ingot_tin"}]; Second: Failed to parse either. First: Not a json array: {"item":"ic2:ingot_tin"}; Second: No key tag in MapLike[{"item":"ic2:ingot_tin"}]; Unknown registry key in ResourceKey[minecraft:root / minecraft:item]: ic2:ingot_tin
```
[latest.log](https://github.com/user-attachments/files/16118961/latest.log)

Both Fabric API and Neoforge provide conditional resource, which allows resource to be parsed only when specific mods are loaded.

**Comment by skniro** [comment]
Port 1.21+ Only.

**Comment by Phoupraw** [comment]
I am exactly playing 1.21

**Comment by skniro** [comment]
Fixed in mod version 1.21-2.3.0(neoforge and fabric).
